src/js/components/DataTableColumns/stories/Simple.js

```diff
@@ -16,6 +16,9 @@ const options = columns.map(({ header, property }) => ({
   label: property === 'name' ? 'Name' : header,
 }));
 
+options[0].pinned = true;
```

Comment:
Instead of this code pattern, can we incorporate the `pinned` logic in the object return of options? Something like:
```
const options = columns.map(({ header, property }, index) => ({
  property,
  label: property === 'name' ? 'Name' : header,
  // demoing pinned behavior
  pinned: index === 0 || index === 1 ? true : undefined,
}));

Also, adjusting to the first 2 columns to be a bit more "realistic" to common use cases

Reply:
to add to this can we revert `Simple` story back and have a new story named Pinned

Reply:
done

Reply:
@edwardhpe It looks like there are still changes in the "Simple" story with the addition of `.pinned` -- can you remove those lines so Simple story is unchanged?